xmtp/src/conversation.rs

```diff
@@ -152,8 +175,16 @@ where
                 )
                 .await
                 .map_err(|_| ConversationError::Unknown)?;
+
+            session.save(conn)?;
         }
 
+        self.client.store.set_conversation_state(
+            conn,
+            self.convo_id().as_str(),
+            ConversationState::Invited,
+        )?;
```

Comment:
Calling out for future discussions: Conversation State is tracked at the peer_address layer, however the messages are fanned out at the installation_id level. This means that any installation that registers after this function was called would not have access to the invite.

Reply:
Ya. Definitely an issue we'll have to solve

Reply:
On the positive side: A newly registered installation will be able to send out its own invite thought which would allow them to join in -- Convo_id's are deterministic so it should just work